doodle-trace/LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2016, Gluon
```

Comment:
It is sufficient to have just one LICENSE file at the root of the repository. Can you move it?